packages/web/src/pages/chat-page/components/ChatHeader.tsx

```diff
@@ -53,7 +58,7 @@ export const ChatHeader = forwardRef<HTMLDivElement, ChatHeaderProps>(
     ref
   ) => {
     const dispatch = useDispatch()
-    const [, setCreateChatVisible] = useModalState('CreateChat')
+    const { onOpen: openCreateChatModal } = useCreateChatModal()
```

Comment:
it's a little weird that this is called `onOpen` versus `open`

Reply:
I feel the same way about `onClose`, should I change them to `open` and `close`?

Reply:
ye i think so, but ok if in sep pr for me